kedro_mlflow/pipeline/pipeline_ml.py

```diff
@@ -37,11 +37,34 @@ def __init__(
         tags: Union[str, Iterable[str]] = None,
         inference: Pipeline,
         input_name: str,
+        conda_env: Union[str, Path, Dict[str, Any]] = None,
+        model_name: Union[str, None] = "model",
     ):
+        """Store all necessary information for calling mlflow.log_model in the pipeline.
+
+        Args:
+            nodes (Iterable[Union[Node,): The `node`s of the training pipeline.
```

Comment:
- [ ] The type of nodes needs updating.
- [ ] The order of arguments in docstring is wrong.

Reply:
I did not notice that the autodocstring extension changed the order of the argument in the docstring. Idon't know why, maybe because of *args not being the last argument.